integr8ly/installation
# INTLY-774 Use AMQ Online 1.0 and remove unused files

Update the EnMasse installer to use AMQ Online 1.0 when the
productized flag is set (which it is by default) and remove all
unused patches that were needed for pre-1.0 versions.

Verification:
- Remove any existing EnMasse namespace
- Ensure there are no enmasse-* files or dirs in /tmp
- Run the installer
- Ensure the EnMasse namespace came up ok
- Go through Walkthrough 1A
